.github/workflows/tilt.yaml

```diff
@@ -0,0 +1,48 @@
+on:
+  pull_request:
+    paths:
```

Comment:
Maybe `paths-ignore` is more suitable here?

Reply:
I kept going back and forth on this. I can switch it if you'd prefer.